src/main/java/main/view/Project/AssignSuiteServlet.java

```diff
@@ -33,7 +33,7 @@ public void doDelete(HttpServletRequest req, HttpServletResponse resp){
             test2Suite.setSuite_id(Integer.parseInt(req.getParameter("suiteId")));
             test2Suite.setTest_id(Integer.parseInt(req.getParameter("testId")));
             session.controllerFactory.getHandler(test2Suite).delete(test2Suite);
-        } catch (AqualityException e) {
+        } catch (Exception e) {
```

Comment:
do we really want to catch any exceptions?

Reply:
The Exception will be handled by handleException method